### PR TITLE
feature: added rules page

### PR DESCRIPTION
**Pull request by omgaikwad**
feature: added rules page
Preview: https://deploy-preview-4--gallant-pasteur-062544.netlify.app/pages/rules/rules.html

### CODE REVIEW

File: pages/question/question.css [hunk 1]
```diff
@@ -39,3 +39,6 @@
 .wrong-option {
   background-color: var(--danger-color);
 }
+.rules-book a {
+  color: white;
```
Comment (RakshithBhat03) [inline pages/question/question.css]: You can use the `--white` variable for this. You have used this variable in rules.css.

File: pages/rules/rules.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.rules-container {
+  list-style-type: disc;
+  color: var(--white);
+  /* border: 1px solid white; */
```
Comment (RakshithBhat03) [inline pages/rules/rules.css]: You can remove this comment.

**Review by RakshithBhat03 — APPROVED** [review]
Everything looks good. The code looks clean and well structured. Good to merge after implementing the suggestions.

**Review by Kothalkarpriya — APPROVED** [review]
Code Looks good to me ! Make changes suggested by Rakshith

File: pages/question/question.css (hunk 1 above)
Comment (omgaikwad) [reply pages/question/question.css]: I will change it!

File: pages/rules/rules.css (hunk 2 above)
Comment (omgaikwad) [reply pages/rules/rules.css]: Sure!

**Pull request merged**